components/dashboard/src/contexts/FeatureFlagContext.tsx

```diff
@@ -4,43 +4,19 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import React, { createContext, useContext, useState, useEffect } from "react";
-import { getExperimentsClient } from "../experiments/client";
+import React, { createContext, useContext, useEffect } from "react";
 import { UserContext } from "../user-context";
 
-const FeatureFlagContext = createContext<{
-    isBlockedRepositoriesUIEnabled: boolean;
-    setIsBlockedRepositoriesUIEnabled: React.Dispatch<boolean>;
-}>({
-    isBlockedRepositoriesUIEnabled: false,
-    setIsBlockedRepositoriesUIEnabled: () => {},
-});
+const FeatureFlagContext = createContext({});
 
 const FeatureFlagContextProvider: React.FC = ({ children }) => {
-    const [isBlockedRepositoriesUIEnabled, setIsBlockedRepositoriesUIEnabled] = useState(false);
     const { user } = useContext(UserContext);
 
     useEffect(() => {
-        (async () => {
-            const isBlockedRepositoriesUIEnabled = await getExperimentsClient().getValueAsync(
-                "isblockedrepositoriesuienabled",
-                false,
-                { user },
-            );
-            setIsBlockedRepositoriesUIEnabled(isBlockedRepositoriesUIEnabled);
-        })();
+        // Query FeatureFlag here
```

Comment:
Do we also need to delete the feature flag from configcat itself?

Reply:
> Do we also need to delete the feature flag from configcat itself?

Jep, once this is deployed. :+1: